ROMFS/px4fmu_common/init.d/airframes/4016_holybro_px4vision

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
+#
+# @name PX4 Vision DevKit Platform
+#
+# @type Quadrotor x
+# @class Copter
+#
+# @output MAIN1 motor 1
+# @output MAIN2 motor 2
+# @output MAIN3 motor 3
+# @output MAIN4 motor 4
+#
+# @board px4_fmu-v2 exclude
+#
+
+sh /etc/init.d/rc.mc_defaults
+
+set MIXER quad_x
+set PWM_OUT 1234
+
+if [ $AUTOCNF = yes ]
+then
+	# System parameters
+	# use FMU motor outputs for less delay in the rate control loop
+	param set SYS_USE_IO 0
```

Comment:
I saw the parameter is using FMU PWM is instead of IO PWM here, but holybro is still using IO PWM on shipping verision now. Looks they have an old customized version. I think we should notify Holybro and early adopters.